src/js/actions/AppsActions.js

```diff
@@ -111,7 +111,7 @@ var AppsActions = {
       headers: {
         "Content-Type": "application/json"
       },
-      url: `${config.apiURL}v2/apps/${appId}`
+      url: `${config.apiURL}v2/apps/${appId}?force=${force}`
```

Comment:
Let's clean up the url if force is false to avoid disfunctionality in the future.

```javascript
let url = !force
  ? `${config.apiURL}v2/apps/${appId}
  : `${config.apiURL}v2/apps/${appId}?force=${force}`;
```

```
url: url
```